### PR TITLE
The shuttle can now be recalled for longer while at green alert

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
tweak: When at green alert, the shuttle can be recalled until the time remaining is less than 5 minutes, from less than 10.
wip: For reference, this means changing the alert to green allows you to recall until the time is less than 2 minutes and 30 seconds at blue alert, and 1 minute and 15 seconds at red and delta alerts.
/:cl:

hey look it's that thing

### CODE REVIEW

File: code/controllers/subsystem/shuttles.dm [hunk 1]
```diff
@@ -238,7 +238,7 @@ var/datum/subsystem/shuttle/SSshuttle
 	var/security_num = seclevel2num(get_security_level())
 	switch(security_num)
 		if(SEC_LEVEL_GREEN)
-			if(emergency.timeLeft(1) < emergencyCallTime)
+			if(emergency.timeLeft(1) < emergencyCallTime * 0.5) //allow recalling for longer when at green alert
 				return
 		if(SEC_LEVEL_BLUE)
 			if(emergency.timeLeft(1) < emergencyCallTime * 0.5)
```
Comment (RemieRichards) [inline code/controllers/subsystem/shuttles.dm]: these are identical now, combine them as ```if(SEC_LEVEL_GREEN, SEC_LEVEL_BLUE)```

**Comment by coiax** [comment]
Control of shuttle recalls also gives complete control of alert levels. Remove the special casing from Green Alert and just lower the recall threshold if that's your intent.

**Comment by KorPhaeron** [comment]
Yeah, this seems like a pretty arbitrary way to obfuscate shuttle recall timers when anyone who can recall a shuttle can also declare a green alert on their own 

Security having absolutely no desire to engage with conversion modes won't change with this though, they'll just smash the consoles

**Comment by ChangelingRain** [comment]
rip design issue why did we make switching alert levels really easy anyway

**Pull request closed without merging**